### DISCUSSION TITLE
v0.5.0

### DISCUSSION THREAD

**Discussion by achimnol** [aio-libs/aiomonitor#369] [Announcements]
## What's Changed
* Scheduled weekly dependency update for week 44 by @pyup-bot in https://github.com/aio-libs/aiomonitor/pull/214
* Try python3.8 by @jettify in https://github.com/aio-libs/aiomonitor/pull/215
* remove unused check by @jettify in https://github.com/aio-libs/aiomonitor/pull/216
* Fix one typo, propose some re-wording in the README file by @kinow in https://github.com/aio-libs/aiomonitor/pull/252
* Bump aiohttp from 3.6.2 to 3.8.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/273
* Update requirements for newer Python versions and use setuptools_scm for project version control by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/332
* Add issue templates by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/331
* remove loop=loop kwarg to support newer Python versions by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/329
* Backport aiomonitor-ng by @achimnol in https://github.com/aio-libs/aiomonitor/pull/334
* Rename refs from aiomonitor-ng to aiomonitor by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/335
* Fix reference to changelog file in setup.py by @rohitsanj in https://github.com/aio-libs/aiomonitor/pull/336
* Make type-anno compatible with Python 3.8 by @achimnol in https://github.com/aio-libs/aiomonitor/pull/338
* Update workflows to run on GitHub by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/328
* Split out the linters from the main ci-cd workflow by @achimnol in https://github.com/aio-libs/aiomonitor/pull/350
* Migrate to pyproject.toml by @achimnol in https://github.com/aio-libs/aiomonitor/pull/354
* Add badges to README by @achimnol in https://github.com/aio-libs/aiomonitor/pull/355
* Migrate the docs to readthedocs by @achimnol in https://github.com/aio-libs/aiomonitor/pull/351
* Rewrite the test suite to work again by @achimnol in https://github.com/aio-libs/aiomonitor/pull/356
* Bump ipdb from 0.13.11 to 0.13.13 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/349
* Bump mypy from 1.0.1 to 1.2.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/348
* Bump aiotools from 1.5.9 to 1.6.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/347
* Bump pytest from 7.1.2 to 7.3.1 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/341
* Bump aioconsole from 0.6.0 to 0.6.1 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/345
* Update sphinx requirement from <6.0.0 to <7.0.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/340
* Bump pytest-sugar from 0.9.6 to 0.9.7 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/343
* Bump attrs from 22.2.0 to 23.1.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/344
* Bump flake8-bugbear from 23.2.13 to 23.3.23 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/342
* Bump isort from 5.10.1 to 5.12.0 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/346
* Let the pytest CI job to collect the code coverage data by @achimnol in https://github.com/aio-libs/aiomonitor/pull/357
* Split out setup.cfg from pyproject.toml by @achimnol in https://github.com/aio-libs/aiomonitor/pull/358
* Let the main CI workflow invoke the lint workflow by @achimnol in https://github.com/aio-libs/aiomonitor/pull/359
* Pass name: ${{ env.sdist-artifact }} while uploading to PyPi by @wbarnha in https://github.com/aio-libs/aiomonitor/pull/361
* Adopt pre-commit-ci for linting and typecheck by @achimnol in https://github.com/aio-libs/aiomonitor/pull/364
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/aio-libs/aiomonitor/pull/366
*  Set up secretless publishing to PyPI by @webknjaz in https://github.com/aio-libs/aiomonitor/pull/367
* Bump aiohttp from 3.8.4 to 3.8.5 by @dependabot in https://github.com/aio-libs/aiomonitor/pull/368

## New Contributors
* @kinow made their first contribution in https://github.com/aio-libs/aiomonitor/pull/252
* @dependabot made their first contribution in https://github.com/aio-libs/aiomonitor/pull/273
* @wbarnha made their first contribution in https://github.com/aio-libs/aiomonitor/pull/332
* @achimnol made their first contribution in https://github.com/aio-libs/aiomonitor/pull/334
* @rohitsanj made their first contribution in https://github.com/aio-libs/aiomonitor/pull/336
* @pre-commit-ci made their first contribution in https://github.com/aio-libs/aiomonitor/pull/366
* @webknjaz made their first contribution in https://github.com/aio-libs/aiomonitor/pull/367

**Full Changelog**: https://github.com/aio-libs/aiomonitor/compare/v0.4.5...v0.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/aio-libs/aiomonitor/releases/tag/v0.5.0'>v0.5.0</a>.</em>

**Comment by wbarnha** [comment]
Thanks for reviving the project!
